sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -523,7 +523,8 @@ static status_t personalize_endorse_certificates(ujson_t *uj) {
  */
 extern status_t personalize_extension_pre_cert_endorse(
```

Comment:
this is a question for @timothytrippel I guess - why are these externs not in a .h file? 

I got burnt by the linker happily linking external modules which had these functions but a misplaced parameter order.

Reply:
Unfortunately, this was a result of bit rot / several refactorings. I agree, we can move this to a header file, but that can be done in a follow up PR.